### ISSUE TITLE
React: Super Sampler did not start correctly. Resetting

### ISSUE THREAD

**Issue by B4TT3RY** [VolmitSoftware/React#222]
Error Log: http://paste.volmit.com/teqoyeyuxi.md

I loaded it with Plugman.

**Comment by lucas2107** [comment]
Is it happening everytime you're loading R6 with Plugman?

**Comment by B4TT3RY** [comment]
yes

**Comment by lucas2107** [comment]
Server Version please? Can't reproduce in 1.8.8 http://prntscr.com/iog3u7

**Comment by B4TT3RY** [comment]
![Image](https://i.imgur.com/sl7Ax34.png)

**Comment by lucas2107** [comment]
Try updating to the latest Paper version (1342)
Can't reproduce: http://prntscr.com/iog5sd

**Comment by B4TT3RY** [comment]
hmm... test server (same main plugins) running.
same error but it's working... (1339 build, 1342 build)

**Comment by B4TT3RY** [comment]
i don't know why my main server does this.

**Comment by Sikatsu** [comment]
For me it seems React 6 doesn't support plugman, it worked with React 5 however.

**Comment by Rbot48** [comment]
I'm also getting this at startup:
```
[06:00:15] [Server thread/INFO]: Starting minecraft server version 1.10.2
[06:00:15] [Server thread/INFO]: Loading properties
[06:00:15] [Server thread/WARN]: resource-pack-hash is deprecated and found along side resource-pack-sha1. resource-pack-hash will be ignored.
[06:00:15] [Server thread/INFO]: Default game type: SURVIVAL
[06:00:15] [Server thread/INFO]: This server is running Paper version git-Paper-916.2 (MC: 1.10.2) (Implementing API version 1.10.2-R0.1-SNAPSHOT)
[06:00:15] [Server thread/INFO]: Debug logging is disabled
[06:00:15] [Server thread/INFO]: Server Ping Player Sample Count: 12
[06:00:15] [Server thread/INFO]: Using 6 threads for Netty based IO
[06:00:15] [Server thread/INFO]: Generating keypair
[06:00:15] [Server thread/INFO]: Starting Minecraft server on []
[06:00:15] [Server thread/INFO]: Using epoll channel type
[06:00:15] [Server thread/INFO]: Set PluginClassLoader as parallel capable
...
[06:00:16] [Server thread/INFO]: [React] Loading React v6.0.9
...
[06:00:18] [Server thread/INFO]: [React] Enabling React v6.0.9
[06:00:19] [Server thread/INFO]: React: Language: enUS
...
[06:01:57] [Server thread/INFO]: React: Super Sampler did not start correctly. Resetting
[06:01:57] [Server thread/INFO]: [React] Disabling React v6.0.9
[06:01:57] [Server thread/INFO]: [React] Enabling React v6.0.9
[06:01:58] [Server thread/INFO]: React: Language: enUS
[06:01:59] [Server thread/INFO]: React: Super Sampler did not start correctly. Resetting
[06:01:59] [Server thread/INFO]: [React] Disabling React v6.0.9
[06:01:59] [Server thread/INFO]: [React] Enabling React v6.0.9
[06:01:59] [Server thread/INFO]: React: Language: enUS
[06:02:00] [Server thread/INFO]: React: Super Sampler did not start correctly. Resetting
[06:02:00] [Server thread/INFO]: [React] Disabling React v6.0.9
[06:02:00] [Server thread/INFO]: [React] Enabling React v6.0.9
[06:02:01] [Server thread/INFO]: React: Language: enUS
[06:02:01] [Server thread/INFO]: React: Super Sampler did not start correctly. Resetting
[06:02:01] [Server thread/INFO]: [React] Disabling React v6.0.9
[06:02:01] [Server thread/INFO]: [React] Enabling React v6.0.9
[06:02:02] [Server thread/INFO]: React: Language: enUS
```
Running PaperSpigot 1.10.2

**Comment by cyberpwnn** [comment]
Fixing #223 will fix this FYI

**Comment by Darkfield** [comment]
Haven't been fixed 6.0.6

**Comment by cyberpwnn** [comment]
Are you replacing the jar on a running vm? Ehem /plugman or something like that? Because this only happens when react was reloaded in a weird way.

**Comment by Darkfield** [comment]
With /plugman load React

**Comment by cyberpwnn** [comment]
Loading after its been loaded is causing it.

**Comment by cyberpwnn** [comment]
Fixed in 6.2.1 If it does fail, it ACTUALLY fixes itself this time.
